vyper/codegen/module.py

```diff
@@ -100,6 +105,10 @@ def _runtime_ir(runtime_functions, all_sigs, global_ctx):
     internal_functions = [f for f in runtime_functions if _is_internal(f)]
 
     external_functions = [f for f in runtime_functions if not _is_internal(f)]
+
+    # all functions with a zero selector, e.g. 0x00000000: changed location and functions referenced
```

Comment:
since we aren't actually using these functions, we can have this be a condition like `has_zero_selector_functions = any(is_zero_selector_func(f) for f in external_functions)`.

Reply:
awesome!